### ISSUE TITLE
Playback freezes on a mpeg4 (Simple Profile) video file.

### ISSUE THREAD

**Issue by sebma** [mpv-player/mpv#3106]
Hi,

I've recorded a video using a smartphone (can't remember which one :) ).

When I try and play the video file, MPV freezes (the video looks paused) but it is not (otherwise, it would mentioned on the terminal) : 

```
$ mpv --no-resume-playback 20141011_012.mp4
Playing: 20141011_012.mp4
 (+) Video --vid=1 (*) (mpeg4)
 (+) Audio --aid=1 --alang=eng (*) (aac)
VO: [opengl] 1920x1088 yuv420p
```
Here is the mediainfo output of this file : 

```
$ mediainfo 20141011_012.mp4
General
Complete name                            : 20141011_012.mp4
Format                                   : MPEG-4
Format profile                           : QuickTime
Codec ID                                 : qt   2005.03 (qt  )
File size                                : 79.4 MiB
Duration                                 : 58s 538ms
Overall bit rate mode                    : Variable
Overall bit rate                         : 11.4 Mbps
Encoded date                             : UTC 2014-10-10 22:31:24
Tagged date                              : UTC 2014-10-10 22:31:24

Video
ID                                       : 1
Format                                   : MPEG-4 Visual
Format profile                           : Simple@L6
Format settings, BVOP                    : Yes
Format settings, QPel                    : No
Format settings, GMC                     : No warppoints
Format settings, Matrix                  : Default (H.263)
Codec ID                                 : 20
Duration                                 : 58s 499ms
Bit rate mode                            : Constant
Bit rate                                 : 11.2 Mbps
Width                                    : 1 920 pixels
Height                                   : 1 088 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Variable
Frame rate                               : 28.034 fps
Minimum frame rate                       : 16.216 fps
Maximum frame rate                       : 34.091 fps
Color space                              : YUV
Bit depth                                : 8 bits
Scan type                                : Progressive
Compression mode                         : Lossy
Bits/(Pixel*Frame)                       : 0.192
Stream size                              : 78.4 MiB (99%)
Language                                 : English
Encoded date                             : UTC 2014-10-10 22:31:24
Tagged date                              : UTC 2014-10-10 22:31:24

Audio
ID                                       : 2
Format                                   : AAC
Format/Info                              : Advanced Audio Codec
Format profile                           : LC
Codec ID                                 : 40
Duration                                 : 58s 538ms
Bit rate mode                            : Variable
Bit rate                                 : 132 Kbps
Maximum bit rate                         : 168 Kbps
Channel(s)                               : 2 channels
Channel positions                        : Front: L R
Sampling rate                            : 48.0 KHz
Frame rate                               : 46.875 fps (1024 spf)
Compression mode                         : Lossy
Stream size                              : 945 KiB (1%)
Language                                 : English
Encoded date                             : UTC 2014-10-10 22:31:24
Tagged date                              : UTC 2014-10-10 22:31:24
```

Can you help me ?

**Comment by wm4** [comment]
Can you upload the file somewhere?

**Comment by sebma** [comment]
The file can be played with `vlc`, `mplayer` and `ffplay`.

**Comment by sebma** [comment]
Ok preparing a share.

**Comment by sebma** [comment]
15 minutes of upload remaining :(

Once you have it, please let me know so I can remove the share.

**Comment by haasn** [comment]
A short, few-second clip would have sufficed.

**Comment by sebma** [comment]
It last 58 seconds

**Comment by sebma** [comment]
If I remux the whole or split it with `ffmpeg`, the resulting file can be played so the pb. cannot be reproduced.

That's why you need the whole file (80MiB).

My upload connection to Dropbox is not very fast.

**Comment by sebma** [comment]
There it is : https://www.dropbox.com/s/rh5p2hb4ftp4sd3/20141011_012.mp4?dl=1

Please let me know once you have finished downloading it so I can remove the share because this video is not public.

**Comment by wm4** [comment]
I've got the file. Works fine with older mpv builds/ffmpeg, freezes with newer mpv/ffmpeg.

**Comment by sebma** [comment]
Ok, removing the share...

**Comment by haasn** [comment]
Works with `--no-audio` for me. Freezes with `--no-video`.

**Comment by sebma** [comment]
Works with `--no-audio` for me as well.

Works with `--no-video`.

Freezes when played normally.

Works if remuxed with ffmpeg : 

`$ ffmpeg -i 20141010_010.mp4 -c copy 20141010_010_FIXED.mp4`

I'm using mpv v0.17 on Ubuntu.

**Comment by wm4** [comment]
It's oddly caused by the new decode API.

**Comment by sebma** [comment]
Hi,

Is this file enough for you to debug mpv ?

Can I remux and delete the bad files because they are much longer and eat up a lot of spaced ?

**Comment by wm4** [comment]
>Is this file enough for you to debug mpv ?

Yes. Btw. what produced this file?

**Comment by sebma** [comment]
I don't know for sure let me try with my actual cell phone.

**Comment by sebma** [comment]
Btw, why this question ?

**Comment by wm4** [comment]
> @wm4 Btw, why this question ?

Because the file is semi-broken. That's why it triggers an unusual corner case.

**Comment by sebma** [comment]
Hi,

Is your fix included in mpv 0.18 ?
